### PR TITLE
Adding Dogstatsd telemetry doc for Ruby

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Update the ruby section of the dogstatsd telemetry documentation.

### Preview link

https://docs-staging.datadoghq.com/maxime/add-statsd-telemetry-doc-ruby

### Additional Notes

This must be merged after https://github.com/DataDog/dogstatsd-ruby/pull/128

### CODE REVIEW

File: content/en/developers/dogstatsd/high_throughput.md [hunk 1]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+
+If you want to disable telemetry, simply do:
+
+```ruby
+Datadog::Statsd.new('localhost', 8125, disable_telemetry: true)
+```
+
+See [DataDog/dogstatsd-ruby](https://github.com/DataDog/dogstatsd-ruby) for more information about the client configuration.
+
 Telemetry will soon be added to the Ruby client.
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]: This line should be removed now that it's available :)

File: content/en/developers/dogstatsd/high_throughput.md [hunk 2]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.metrics`         | count       | Number of `metrics` sent to the client by your application (before sampling).             |
+| `datadog.dogstatsd.client.events`          | count       | Number of `events` sent to the client by your application.                                |
+| `datadog.dogstatsd.client.service_checks`  | count       | Number of `service_checks` sent to the client by your application.                        |
+| `datadog.dogstatsd.client.bytes_sent`      | count       | Number of bytes successfully sent to the Agent.                                           |
+| `datadog.dogstatsd.client.bytes_dropped`   | count       | Number of bytes dropped by the client.                                                    |
+| `datadog.dogstatsd.client.packets_sent`    | count       | Number of datagrams successfully sent to the agent.                                       |
+| `datadog.dogstatsd.client.packets_dropped` | count       | Number of datagrams dropped by the client.                                                |
+
+If you want to disable telemetry, simply do:
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
To disable telemetry, set the `disable_telemetry` parameter to `true`:
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 3]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
 {{% /tab %}}
 {{% tab "Ruby" %}}
 
+Starting with version `4.6.0` of the Ruby client.
+
+| Metrics Name                               | Metric Type | Description                                                                               |
+|--------------------------------------------|-------------|-------------------------------------------------------------------------------------------|
+| `datadog.dogstatsd.client.metrics`         | count       | Number of `metrics` sent to the client by your application (before sampling).             |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.metrics`         | count       | Number of `metrics` sent to the DogStatsD client by your application (before sampling).             |
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 4]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.events`          | count       | Number of `events` sent to the client by your application.                                |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.events`          | count       | Number of `events` sent to the DogStatsD client by your application.                                |
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 5]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.service_checks`  | count       | Number of `service_checks` sent to the client by your application.                        |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.service_checks`  | count       | Number of `service_checks` sent to the DogStatsD client by your application.                        |
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 6]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.bytes_sent`      | count       | Number of bytes successfully sent to the Agent.                                           |
+| `datadog.dogstatsd.client.bytes_dropped`   | count       | Number of bytes dropped by the client.                                                    |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.bytes_dropped`   | count       | Number of bytes dropped by the DogStatsD client.                                                    |
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 7]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.packets_sent`    | count       | Number of datagrams successfully sent to the agent.                                       |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.packets_sent`    | count       | Number of datagrams successfully sent to the Agent.                                       |
```

File: content/en/developers/dogstatsd/high_throughput.md [hunk 8]
```diff
@@ -256,6 +256,26 @@ See [DataDog/datadogpy](https://github.com/DataDog/datadogpy) for more informati
...
+| `datadog.dogstatsd.client.packets_dropped` | count       | Number of datagrams dropped by the client.                                                |
```
Comment (l0k0ms) [inline content/en/developers/dogstatsd/high_throughput.md]:
```suggestion
| `datadog.dogstatsd.client.packets_dropped` | count       | Number of datagrams dropped by the DogStatsD client.                                                |
```

**Comment by hush-hush** [comment]
@l0k0ms I also updated the Go and Python doc to follow you comment on Ruby.

**Comment by l0k0ms** [comment]
ha indeed ! i had open another PR to fix this but it's perfect if it's done here :) many thanks for the stretch !

**Pull request merged**